### PR TITLE
Speakerphone

### PR DESCRIPTION
**Pull request by wbarnard**
Adds support for speakerphone feature in Android/iOS.

### CODE REVIEW

**Comment by cloudwebrtc** [comment]
@wbarnard File conflict after merging https://github.com/cloudwebrtc/flutter-webrtc/pull/101.

File: android/src/main/java/com/cloudwebrtc/webrtc/FlutterWebRTCPlugin.java [hunk 1]
```diff
@@ -894,11 +900,21 @@ public void mediaStreamTrackSetVolume(final String id, final double volume) {
         }
     }
 
+<<<<<<< HEAD
```
Comment (cloudwebrtc) [inline android/src/main/java/com/cloudwebrtc/webrtc/FlutterWebRTCPlugin.java]: Here is a conflicting diff tag, which causes compilation errors.

**Comment by wbarnard** [comment]
@cloudwebrtc rebased to fix conflict

File: android/src/main/java/com/cloudwebrtc/webrtc/FlutterWebRTCPlugin.java [hunk 2]
```diff
@@ -894,11 +900,21 @@ public void mediaStreamTrackSetVolume(final String id, final double volume) {
...
     public void mediaStreamTrackSetMicrophoneMute(final String id, boolean mute) {
         try {
             audioDeviceModule.setMicrophoneMute(mute);
         } catch (Exception e) {
             Log.e(TAG, "setMicrophoneMute(): error", e);
+=======
```
Comment (cloudwebrtc) [inline android/src/main/java/com/cloudwebrtc/webrtc/FlutterWebRTCPlugin.java]: The same question as above, it looks like need to add
```dart
        }
    }
```

File: android/src/main/java/com/cloudwebrtc/webrtc/FlutterWebRTCPlugin.java [hunk 3]
```diff
@@ -894,11 +900,21 @@ public void mediaStreamTrackSetVolume(final String id, final double volume) {
...
+    public void mediaStreamTrackEnableSpeakerphone(final String id, boolean enabled) {
+        AudioManager audioManager = (AudioManager) getContext().getSystemService(Context.AUDIO_SERVICE);
+
+        try {
+            audioManager.setSpeakerphoneOn(enabled);
+        } catch (Exception e) {
+            Log.e(TAG, "setSpeakerphoneOn(): error", e);
+>>>>>>> Add speakerphone support for Android
```
Comment (cloudwebrtc) [inline android/src/main/java/com/cloudwebrtc/webrtc/FlutterWebRTCPlugin.java]: This is a conflict marker and should be removed.

**Comment by wbarnard** [comment]
@cloudwebrtc rebased again to fix conflict

**Pull request merged**